platform/macos/src/MGLMapView.h

```diff
@@ -247,7 +247,8 @@ MGL_EXPORT IB_DESIGNABLE
  If the value of this property is smaller than that of the `minimumZoomLevel`
  property, the behavior is undefined.
 
- The default value of this property is 20.
+ The default value of this property is 22. The upper bound for this property
+ is 25.5.
```

Comment:
[The corresponding iOS comment](https://github.com/mapbox/mapbox-gl-native/blob/bd15e273dce767458d335aeb1f50aa081390d593/platform/ios/src/MGLMapView.h#L600) needs to be updated.